content/en/docs/tutorials/configuration/kubelet-standalone.md

```diff
@@ -0,0 +1,393 @@
+---
+title: Running Kubelet in Standalone Mode
+content_type: tutorial
+weight: 5
+card:
+  name: tutorials
+  weight: 10
+---
+
+<!-- overview -->
+
+This tutorial shows you how to run a standalone Kubelet instance.
+
+In standalone mode the Kubelet allows you to manage containers using pod manifests, which brings the benefits of running tightly coupled application as a single unit while leveraging a subset of advanced features including init containers, CNI networking, etc., without the need for the entire Kubernetes feature set.
+
+## {{% heading "objectives" %}}
+
+* Install and configure `containerd`, `cni-plugins`, and `kubelet` on a Linux system.
+* Run `containerd` and `kubelet` as `systemd` services.
+* Launch a Pod running `nginx` that listens to requests on TCP port 80 on the Pod's IP address.
+
+## {{% heading "prerequisites" %}}
+
```

Comment:
```suggestion

You need a computer with an AMD64 or compatible processor. This tutorial does not provide
instructions for other architectures.
You also need:

```
(if we could cover, eg, ARM64 as well, that would be nice)

Reply:
I don't have access to an ARM64 system to test but I added the `arm64` download URLs for all the components and duplicated the download and the install commands in tabs format for both `amd64` and `arm64`.

Reply:
BTW, you can use (eg) qemu to test locally without buying new equipment. I think the testing you said you did is enough though.